library/Fum2GitHub/Fum.hs

```diff
@@ -1,89 +1,67 @@
 {-# LANGUAGE OverloadedStrings #-}
 module Fum2GitHub.Fum (
     getAllUsers,
-    User(User), username, github,
-    userFromAPI,
+    User(..),
+    UsersResult(..)
 ) where
 
-import           Control.Applicative ((<*>), (<$>))
-import           Data.Aeson ((.:))
-import qualified Data.Aeson as Aeson
-import           Data.Aeson.Types (parseEither)
+import           Control.Applicative
```

Comment:
We'd prefer to keep imports explicit so when we see `someName` in the source we know where it comes from.